cloud/cmd/csidriver/app/server_test.go

```diff
@@ -63,8 +63,8 @@ func TestNewCSIDriverCommand(t *testing.T) {
 
 	versionFlag := flags.Lookup("version")
 	assert.NotNil(versionFlag)
-	assert.Equal("version", versionFlag.Usage)
-	assert.Equal("dev", versionFlag.DefValue)
+	assert.Equal("Print version information and quit", versionFlag.Usage)
+	assert.Equal("false", versionFlag.DefValue)
```

Comment:
Looks like we've deleted the version flag, why versionFlag is still tested here?

Reply:
actually, we just deleted the duplicate `version flag` definition, we still have the version flag and its been added [here](https://github.com/kubeedge/kubeedge/blob/e60886d889b26a4d9e8886991475bdf6ba0423f6/cloud/cmd/csidriver/app/server.go#L61). The duplicate flag was causing inconsistency during CI tests, now only one version flag definition is there.

Reply:
looks good, thanks!